app/src/main-process/main.ts

```diff
@@ -531,6 +531,20 @@ app.on('ready', () => {
       UNSAFE_openDirectory(path)
     }
   })
+
+  /**
+   * An event sent by the renderer asking to show the open dialog
+   */
+  ipcMain.handle(
+    'show-open-dialog',
+    async (_, options: Electron.OpenDialogOptions) => {
+      if (mainWindow === null) {
+        return null
+      }
+
+      return mainWindow.showOpenDialog(options)
+    }
+  )
```

Comment:
Let's see if we can't get this a little bit more succinct

```suggestion
  ipcMain.handle('show-open-dialog', async (_, options) => {
    return mainWindow?.showOpenDialog(options) ?? null
  })
```

Reply:
I will have a pull request for the `show-save-dialog` -> I will update this on that PR since it will follow a similar pattern.